pywbemtools/pywbemlistener/_cmd_listener.py

```diff
@@ -865,7 +892,9 @@ def parse_listener_args(listener_args):
     # Note: The following options must ne in sync with the Click command options
     parser.add_argument('--start-pid', type=int, default=None)
     parser.add_argument('--port', '-p', type=int,
-                        default=DEFAULT_LISTENER_PORT)
+                        default=None)
```

Comment:
I don't understand this change of the port default to `None`. Normally when a default of `None` is set, there is some processing of the value None to handle this case with code, but I don't see that.

Reply:
Actually either works since the default value is set in the click processing but this more clearly makes the point.  Fixed